### DISCUSSION TITLE
Cancelling an Appsource AL Submission

### DISCUSSION THREAD

**Discussion by fibbert** [microsoft/AL-Go#388] [Q&A]
This is probably the wrong forum - and I cross posted it too - but does anyone know how to request an AL submission to Appsource be withdrawn?  We published the wrong version and it will be a breaking change to revert.

**Comment by freddydk** [comment]
Create a support ticket in Business Central

**Reply by laurag-sabrelimited** [reply]
Will that cancel the submission process even if it was in the Certification phase, so pass the Publisher sign off?

**Reply by freddydk** [reply]
No - cancelling the submission can only cancel before you press Go-Live.
You need to create a support ticket in Business Central to get this sorted out - nobody can help you without a support ticket.

**Comment by fibbert** [comment]
Support ticket was created but they’re basically saying what you said. How do we escalate to get them to manually override?

Robert Jolliffe
1-519-223-8634
President, Sabre Limited & MicroAge Kitchener
Book a meeting with Microsoft Bookings:

***@***.***/bookings/

________________________________

**Comment by freddydk** [comment]
You need to remove a version of an app, which is live in AppSource.
You can only do this if nobody installed the app.
They should be able to help with that.

**Reply by laurag-sabrelimited** [reply]
if the offer is "stop publishing", the documentation reads that existing customers won't be affected, but they also won't get any updates once the offer is republished. Does that mean the existing customers won't be able to update their environments once a new version of the offer is published?
